gnomad/resources/__init__.py

```diff
@@ -1 +1,3 @@
+# noqa: D104
```

Comment:
Just asking for my knowledge, is there a reason its better that we specify D104 in each __init__.py rather than add where D105 and D107 are added in `pydocstylerc`?

Reply:
If we ignore it individually in existing files, then pydocstyle will still complain about missing module docstrings in new files. Versus if we ignore it in `pydocstylerc`, new files can be created without module docstrings.